client/src/utils/CSSHelp.test.ts

```diff
@@ -0,0 +1,81 @@
+/* global describe it expect */
+import { cssString } from './__fixtures/curriculum-helpers-css';
+import CSSHelp from './CSSHelp';
+
+describe('CSSHelp should pass all tests..', () => {
+  document.body.innerHTML = '';
+  document.head.innerHTML = '';
+  const doc = document;
+  let t: CSSHelp;
+  beforeAll(() => {
```

Comment:
```suggestion
  beforeEach(() => {
```
In principle (I know they don't) each test could modify the document, so we might as well setup and teardown around each one.

Reply:
Does this come with any penalties?

Reply:
It's a bit slower, but not disastrously slower.